tests/test_bot.py

```diff
@@ -295,10 +295,10 @@ async def shutdown():
 
         assert self.test_flag == "stop"
 
-    async def test_log_decorator(self, bot, caplog):
+    async def test_log_decorator(self, bot: DictExtBot, caplog):
         # Second argument makes sure that we ignore logs from e.g. httpx
         with caplog.at_level(logging.DEBUG, logger="telegram"):
-            await bot.get_me()
+            await super(bot.__class__, bot).get_me()  # call original get_me instead of overriden
```

Comment:
This might be fragile if the inheritance order of `DictExtBot` changes. Please instead just instaniate an instance of `ExtBot` with `bot.token`.